client/connector.go

```diff
@@ -114,3 +123,16 @@ func (c *Client) DeleteConnector(connectorId string, networkItemId string, netwo
 	_, err = c.DoRequest(req)
 	return err
 }
+
+func (c *Client) GetConnectorProfile(id string) (string, error) {
+	req, err := http.NewRequest(http.MethodPost, fmt.Sprintf("%s/api/beta/connectors/%s/profile", c.BaseURL, id), bytes.NewBufferString(""))
```

Comment:
I guess you can just use `nil` instead of `bytes.NewBufferString("")` if the request body is not required.

Reply:
replaced it with nil